tests/test_browser.py

```diff
@@ -2704,81 +2704,82 @@ def test_wget_data(self):
 
   def test_locate_file(self):
```

Comment:
Can you do this instead?  (which should avoid all the re-indentation).

```
  @parameterized({                                                                                      
    '': [false],                                                                               
    'es6': [true]})
```

this will give you two separate tests that can each fail independently.   You can then run just one with `browser.test_locate_file_es6` or both using `*` (`browser.test_locate_file*`).

(One could argue that the wasm variant should also be done this way but in the interested of minimizing the size of this change I suggest you just do it for your new variant).

Reply:
Sigh, so many different styles to be aware of 😀